src/config.rs

```diff
@@ -67,6 +67,16 @@ impl<'de, T: Deserialize<'de>> Deserialize<'de> for Inferrable<T> {
     }
 }
 
+impl<T> Inferrable<T> {
+    pub fn is_none(&self) -> bool {
+        if let &Inferrable::None = self {
+            return false;
+        } else {
+            return true;
+        }
```

Comment:
This would be better as a `match`

Reply:
Done. Let's, uh, not talk about previous, totally correct `return false`, though!